sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -47,364 +48,444 @@ const alwaysBeDeletedSubscription = "alwaysbedeletedsubscription";
+      let response: any;
+
+      response = await updateEntity(
+        testCase.entityType,
+        testCase.alwaysBeExistingEntity,
+        testCase.parentTopicName,
+        testCase.parentSubscriptionName
+      );
+      should.equal(
+        response[testCase.entityType.toLowerCase() + "Name"],
+        testCase.alwaysBeExistingEntity,
+        "Entity name mismatch"
+      );
+    });
+
+    it(`Update on non-existent ${testCase.entityType} entity throws an error`, async () => {
+      let response;
+
+      try {
+        await deleteEntity(
+          testCase.entityType,
+          testCase.alwaysBeDeletedEntity,
+          testCase.parentTopicName,
+          testCase.parentSubscriptionName
+        );
+      } catch (err) {
+        log.httpAtomXml("Ignoring clean up step");
+      }
+      try {
+        await updateEntity(
+          testCase.entityType,
+          testCase.alwaysBeDeletedEntity,
+          testCase.parentTopicName,
+          testCase.parentSubscriptionName
+        );
+      } catch (err) {
+        response = err;
+      }
+
+      should.equal(response.statusCode, 404, "Error must not be undefined");
+      should.equal(
+        response.message.startsWith("The messaging entity") ||
+          response.message.startsWith("Entity") ||
+          response.message.startsWith("SubCode") ||
+          response.message.startsWith("No service"),
+        true,
+        `Expected error message to be a textual content but got "${response.message}"`
+      );
+    });
+
+    it(`Gets an existent ${testCase.entityType} entity successfully`, async () => {
+      let response;
+
+      response = await getEntity(
+        testCase.entityType,
+        testCase.alwaysBeExistingEntity,
+        testCase.parentTopicName,
+        testCase.parentSubscriptionName
+      );
+      should.equal(
+        response[testCase.entityType.toLowerCase() + "Name"],
+        testCase.alwaysBeExistingEntity,
+        "Entity name mismatch"
+      );
+    });
+
+    it(`Deletes a non-existent ${testCase.entityType} entity returns an error`, async () => {
+      let response;
+
+      try {
+        await deleteEntity(
+          testCase.entityType,
+          "notexisting",
+          testCase.parentTopicName,
+          testCase.parentSubscriptionName
+        );
+      } catch (err) {
+        response = err;
+      }
+
+      should.equal(response.statusCode, 404);
+      should.equal(
+        response.message.startsWith("The messaging entity") ||
+          response.message.startsWith("Entity") ||
+          response.message.startsWith("SubCode") ||
+          response.message.startsWith("No service"),
+        true,
+        `Expected error message to be a textual content but got "${response.message}"`
+      );
+    });
+
+    it(`Deletes an existent ${testCase.entityType} entity successfully`, async () => {
+      let response;
+
+      try {
```

Comment:
Same comment about removing try/catch.

Reply:
If a test fails on CI, then manual scripts would need to be run - or the "clean namespace" mechanism we have should be updated to delete all the entities.
For such cases (specifically, simple delete and create tests), had added in the ignorable clean up steps. 
However, updated it now to use isolation as the CI runs are generally expected to be clean